command/kv_patch.go

```diff
@@ -185,7 +185,17 @@ func (c *KVPatchCommand) Run(args []string) int {
 		return code
 	}
 
-	return OutputSecret(c.UI, secret)
+	if Format(c.UI) != "table" {
+		return OutputSecret(c.UI, secret)
```

Comment:
This is an early successful return, which is a bit non-standard in Go. Is it possible to fix this by inverting the `if` to `if Format(c.UI) == "table" {...}`. Alternatively, I would add a comment here: 
```go
return OutputSecret(c.UI, secret) // early return
```

Reply:
There were a few places already in the KV command code that were already doing `return OutputSecret` earlier based on a flag and then having `return 0` at the very end (such as kv get), so I tried to make it work similarly to those. OutputSecret is capable of failing and returning exit code 1 so I think that's the rationale behind it sometimes coming earlier.

Reply:
Ended up switching it around like you'd suggested after all